lighthouse-core/lib/i18n/locales/fr.json

```diff
@@ -447,7 +447,7 @@
     "message": "Valeur"
   },
   "lighthouse-core/audits/dobetterweb/dom-size.js | description": {
-    "message": "Les ingénieurs en navigation recommandent que les pages contiennent moins de 1 500 éléments DOM environ. La zone d'écoute idéale est une profondeur d'arborescence inférieure à 32 éléments et contenant moins de 60 éléments enfant/parent. Un grand DOM peut accroître l'utilisation de la mémoire, entraîner de plus longs [calculs de style](https://developers.google.com/web/fundamentals/performance/rendering/reduce-the-scope-and-complexity-of-style-calculations) et des [redispositions] coûteuses (https://developers.google.com/speed/articles/reflow). [En savoir plus](https://developers.google.com/web/tools/lighthouse/audits/dom-size))."
+    "message": "Les ingénieurs en navigation recommandent que les pages contiennent moins de 1 500 éléments DOM environ. La zone d'écoute idéale est une profondeur d'arborescence inférieure à 32 éléments et contenant moins de 60 éléments enfant/parent. Un grand DOM peut accroître l'utilisation de la mémoire, entraîner de plus longs [calculs de style](https://developers.google.com/web/fundamentals/performance/rendering/reduce-the-scope-and-complexity-of-style-calculations) et des [redispositions] coûteuses (https://developers.google.com/speed/articles/reflow). [En savoir plus](https://developers.google.com/web/tools/lighthouse/audits/dom-size)."
```

Comment:
`[redispositions] coûteuses (https://developers.google.com/speed/articles/reflow).`

"redispositions" is disconnected from its link...and it doesnt seem to play nice with the report...
![image](https://user-images.githubusercontent.com/6392995/58040654-ceaeaa00-7aea-11e9-8692-51406440f7f4.png)

Reply:
Made #9000 to track this.

Reply:
can you fix manually for now? We can't ship this

Reply:
(I'm glad you noticed because I don't think I would have :)

Reply:
Fixed!

![image](https://user-images.githubusercontent.com/6392995/58124571-68498a80-7bc3-11e9-9f5b-73ccc37ebabd.png)